### PR TITLE
AEA-2035 Add prescription signature verification to EPSAT

### PR DESCRIPTION
**Pull request by booshibooshi**

### CODE REVIEW

File: coordinator/src/services/translation/response/release/release-medication-request.ts [hunk 1]
```diff
@@ -122,7 +122,7 @@ function createBasedOn(
     }]
   }
   return [{
-    identifier: reference,
```
Comment (booshibooshi) [inline coordinator/src/services/translation/response/release/release-medication-request.ts:125]: to separate this change from this pr

File: coordinator/src/services/translation/response/organization.ts [hunk 2]
```diff
@@ -60,8 +60,8 @@ function getFixedOrganizationType() {
       coding:  [
         {
           system: "https://fhir.nhs.uk/CodeSystem/organisation-role",
-          code: "197",
-          display: "NHS TRUST"
```
Comment (booshibooshi) [inline coordinator/src/services/translation/response/organization.ts:64]: to separate this change from this pr

File: coordinator/tests/services/translation/response/organization.spec.ts [hunk 3]
```diff
@@ -35,7 +35,7 @@ describe.each([
 
     test("%p has a type block with correct coding values", () => {
       expect(fhirOrganization.type).not.toBeUndefined()
-      expect(fhirOrganization.type[0].coding[0].code).toBe("197")
+      expect(fhirOrganization.type[0].coding[0].code).toBe("198")
       expect(fhirOrganization.type[0].coding[0].display).toBeTruthy()
```
Comment (booshibooshi) [inline coordinator/tests/services/translation/response/organization.spec.ts:39]: to separate this change from this pr

File: coordinator/src/services/translation/response/common.ts [hunk 4]
```diff
@@ -189,39 +189,52 @@ export function translateAgentPerson(agentPerson: hl7V3.AgentPerson): Translated
       locations
     }
   } else {
```
Comment (booshibooshi) [inline coordinator/src/services/translation/response/common.ts:191]: to separate coordinator changes from this pr

File: tool/site/client/src/components/prescription-tracker/prescriptionSummaryList.tsx [hunk 5]
```diff
@@ -88,7 +88,7 @@ export const PrescriptionSummaryList: React.FC<PrescriptionSummaryProps> = ({
             <StyledPrescriptionActions prescriptionId={id} release />
           }
           {status === "With Dispenser" &&
-            <StyledPrescriptionActions prescriptionId={id} dispense releaseReturn />
+            <StyledPrescriptionActions prescriptionId={id} verify dispense releaseReturn />
```
Comment (JackRobinsonIW) [inline tool/site/client/src/components/prescription-tracker/prescriptionSummaryList.tsx:91]: Is there any reason verify is only on this status other than it make the most sense from a business workflow perspective? We could theoretically verify the signature whenever right?

File: tool/site/client/src/pages/loadPage.tsx [hunk 6]
```diff
@@ -122,10 +122,6 @@ const LoadPage: React.FC = () => {
                     value: "secondary-care/community/repeat-dispensing/nominated-pharmacy/clinical-practitioner/single-medication-request",
                     text: "Secondary Care - Repeat Dispensing (nominated)"
                   },
-                  {
-                    value: "secondary-care/homecare/acute/nominated-pharmacy/clinical-practitioner",
-                    text: "Homecare - Acute (nominated)"
-                  },
```
Comment (JackRobinsonIW) [inline tool/site/client/src/pages/loadPage.tsx:128]: Couldn't really see why we've deleted this one

**Review by JackRobinsonIW — APPROVED** [review]

File: tool/site/client/src/pages/loadPage.tsx (hunk 6 above)
Comment (booshibooshi) [reply tool/site/client/src/pages/loadPage.tsx:128]: our first of type users will not be using homecare and we're not confident it's a representative example so dropping until we're asked to make it available and had feedback on what's wrong with it

**Pull request merged**